README.md

```diff
@@ -0,0 +1,95 @@
+Water-Energy-Food-Environment (WEFE) mini-grids. These combine different renewable energy (RE) technologies,
+such as photovoltaics, wind turbines, hydropower, and biogas plants, storage systems such as
+batteries, hydrogen, or water tanks, and productive uses, such as water pumping, water treatment,
+irrigation, cold storage, agricultural processing, via smart control systems. An example of such are agrivoltaics which 
+provide sustainable renewable energy production tapping WEFE synergies (e.g. reducing land competition
+between energy and food production and reducing evapotranspiration from the field, leading to less
+irrigation demand). It utilizes [oemof.solph](https://github.com/oemof/oemof-solph) and 
+[oemof.tabular](https://github.com/oemof/oemof-tabular) libraries, along with additional plugins for extended 
+functionality. Users can create and run scenarios using different technologies as components with their respective 
+demand and generation profiles as [csv](https://en.wikipedia.org/wiki/Comma-separated_values) files. Moreover, the user 
+can also define new components/technologies in addition to the existing ones. The results from the modelled scenario can
+then be interactively visualized through [dash](https://dash.plotly.com/).
+
+## Prerequisites
+
+Make sure a supported Python Version (3.9 or greater) and an interpreter (preferably PyCharm) is installed on your system. 
+Install CBC solver for oemof-solph using the installation instructions: 
+https://oemof-solph.readthedocs.io/en/latest/readme.html#installing-a-solver
+
+## Installation
+
+For using the WEFEConfigurator, clone the repository to your local machine. Then create a new virtual environment with 
+a supported Python Version (3.9 or greater). Activate the new virtual environment and move to the repository folder to 
+install the oemof-tabular-plugins package.
+
+    pip install oemof_tabular_plugins==0.0.2rc4 numpy==1.26.0 git+https://github.com/oemof/oemof-tabular.git@dev git+https://github.com/sedos-project/oemof.industry.git@saltwater
+
+Possible Errors: 
+1) Please copy and paste the whole statement, there are no line skips.
+2) For Windows users, an error might appear when installing oemof-tabular: 
+    
+    "ERROR: Could not build wheels for cchardet, which is required to install pyproject.toml-based projects"
+3) Guidelines on how to fix this error: https://github.com/twintproject/twint/issues/1407#issuecomment-1141734344
+
+Verify the installation of CBC solver by typing in the terminal: 
+
+    oemof_installation_test    
+
+If successful, the following message will appear: ![img.png](img.png)
+
+If unsuccessful, the message will say "cbc: not working", in which case an external executable "cbc.exe" file from a 
+successful installation should be included in your path and your system restarted.
+
+Once all of these steps have been completed successfully, you are ready to run scenarios!
+
+## Get started
+
+
+## Installation
```

Comment:
From here this is part the the template, I would leave it out for the moment :)

Reply:
okay understood